### DISCUSSION TITLE
台積電驚見3字頭 零股周年慶來了？網喊還有得跌：等這價位

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#765] [General]
前幾年台灣半導體興盛時 許多人都紛紛購入台積電的股票 想說可藉由這大好情勢 好好大撈一筆 然而政治緣故 台積電大跌 希望台灣護國山能撐過這次的災情

https://wantrich.chinatimes.com/news/20221012900332-420101

**Comment by danwan1027** [comment]
想當初想要在 600 揀便宜的，現在還被套在山上下不來

**Comment by williamyen0623** [comment]
還敢在600的地方撿便宜啊冰鳥

**Comment by manipulateed** [comment]
買賣股票本來就要慎選考慮未來或是政治等等的發展，沒有永遠的漲跌亦是如此，看太重反而被套牢了。

**Comment by ghost** [comment]
我們應該謹記股神巴菲特說的話：當別人恐懼的時候我貪婪，當別人貪婪的時候，我戒慎恐懼。

**Comment by Pietra1226** [comment]
電子股還是存在很大風險 還是買我的0050好了

**Comment by CPCpyt** [comment]
與其討論被套，在下跌時有沒有加倉回補之類的策略，才會影響到後續是否有獲利。
